travis-update-book.sh

```diff
@@ -56,10 +59,10 @@ ENCRYPTED_KEY_VAR="encrypted_${ENCRYPTION_LABEL}_key"
 ENCRYPTED_IV_VAR="encrypted_${ENCRYPTION_LABEL}_iv"
 ENCRYPTED_KEY=${!ENCRYPTED_KEY_VAR}
 ENCRYPTED_IV=${!ENCRYPTED_IV_VAR}
-openssl aes-256-cbc -K $ENCRYPTED_KEY -iv $ENCRYPTED_IV -in ../../../deploy_key.enc -out ../../../deploy_key -d
-chmod 600 ../../../deploy_key
+openssl aes-256-cbc -K $ENCRYPTED_KEY -iv $ENCRYPTED_IV -in ../../../build_docs_key.enc -out ../../../build_docs_key -d
```

Comment:
I would rather like the keys to be injected via travis env variables

Reply:
Do you mean eliminating the encrypted key? It was encrypted with travis and set up an environment variable which is used to decrypt it. So it seems to me that it is injecting via travis env variables

Reply:
No, I mean you can inject the key directly without encryption via travis. Travis will hide the key form the logs

Reply:
<img width="1083" alt="screen shot 2018-08-08 at 14 46 38" src="https://user-images.githubusercontent.com/3702016/43837793-e511bf8c-9b19-11e8-831a-56bd120d16ad.png">

Reply:
Would you be able to hop on a call briefly today to show me what you mean? I could fumble at trying to ask about it, but I think that might be easier